### PR TITLE
Performance: Run weekly audits against Sourcegraph.com

### PR DESCRIPTION
**Pull request by umpox**
image

The plan is to post this to #dev-frontend once a week

### CODE REVIEW

File: dev/ci/generate-lighthouse-slack-report.sh [hunk 1]
```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
+# This script will run Lighthouse audits against a specified URL
```
Comment (jhchabran) [inline dev/ci/generate-lighthouse-slack-report.sh:3]: Non-blocking comment: we're trying to make sure the new shell scripts are listing the mandatory env variables they rely on, making it easier to maintain them. It seems that here, there are actually no dependencies right?

**Review by jhchabran — APPROVED** [review]
Hey @umpox 👋😊! That's really cool! 

> 1. We can commit the task directly to `main` instead of having to run from a branch, or set up complex logic around an env var to modify the pipeline

I'm not sure to understand the `main` VS `branch` here, would you mind elaborating a bit? On the other hand, with the env vars assignation workflow, I totally see the problem. That being said, making updates to GH secrets requires reaching someone with admin permissions on the repo. 

> Let me know if there's any preference on if this should run through Buildkite!

This task is really self-contained so GH action seems to be a good choice for now.

File: dev/ci/generate-lighthouse-slack-report.sh (hunk 1 above)
Comment (umpox) [reply dev/ci/generate-lighthouse-slack-report.sh:3]: That's right, no dependencies :)

**Comment by umpox** [comment]
@jhchabran Thanks!

I initially set this up through Buildkite by modifying our main pipeline in a branch and then running a scheduled build through on that branch. I guess in this case it'd be easier just to setup a separate pipeline and then it could just run off `main` - not sure how much work that is though

**Pull request merged**